### DISCUSSION TITLE
Please help test version 4.0.0 beta!

### DISCUSSION THREAD

**Discussion by hvianna** [hvianna/audioMotion-analyzer#40] [Announcements]
- See the [latest release notes](https://github.com/hvianna/audioMotion-analyzer/releases/tag/4.0.0-beta.3)
- See the new features in action on this pen: https://codepen.io/hvianna/pen/MWBgjMb
(look for "New in v4.0.0" in the controls)
- Check the [*develop* branch](https://github.com/hvianna/audioMotion-analyzer/tree/develop) for updated documentation
- Install from npm: `npm i audiomotion-analyzer@beta`

**Comment by Staijn1** [comment]
I will update my typescript project somewhere this week, I'll let you know if I run into any issues.
Release 4 is looking pretty sick, thank you so much for your work!

**Reply by Staijn1** [reply]
I've just upgraded my project without issues.

Question: If I select two different gradients for the left and right channel, and channelLayout is **not** single, which background definition is used?
I'd assume the background defined in gradientLeft, because this gradient is also used when channelLayout **is** single, but I couldn't find this in the documentation.

**Reply by hvianna** [reply]
Thanks, appreciate your help!

Good catch on the background color! Currently only the bgcolor defined for gradientLeft is used, but I'll look into using both colors when possible. Thanks again!

**Reply by hvianna** [reply]
As of [beta.5](https://github.com/hvianna/audioMotion-analyzer/discussions/43) both background colors are used in 'dual-vertical' channel layout.
